app/src/main/java/at/roteskreuz/stopcorona/model/workers/ProcessDiagnosisKeysWorker.kt

```diff
@@ -0,0 +1,62 @@
+package at.roteskreuz.stopcorona.model.workers
+
+import android.content.Context
+import androidx.work.*
+import at.roteskreuz.stopcorona.model.exceptions.SilentError
+import at.roteskreuz.stopcorona.model.repositories.InfectionMessengerRepository
+import org.koin.standalone.KoinComponent
+import org.koin.standalone.inject
+import timber.log.Timber
+import kotlin.IllegalArgumentException
+
+/**
+ * Worker that displays local notifications.
```

Comment:
Little bit generic. I guess the worker is more specialised.

Reply:
@falkorichter

Reply:
@falkorichter